hooks/useQueries.ts

```
@@ -750,68 +752,147 @@
+      };
+    case `/osmosis.tokenfactory.v1beta1.MsgBurn`:
+      return {
+        data: {
+          tx_type: HistoryTxType.BURN,
+          from_address: message.sender,
+          to_address: message.burnFromAddress,
+          amount: [message.amount],
+        },
+      };
+    case `/liftedinit.manifest.v1.MsgPayout`:
+      const totalAmountMap = new Map<string, bigint>();
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)